02week/ticTacToe.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const assert = require('assert');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+let board = [
+  [' ', ' ', ' '],
+  [' ', ' ', ' '],
+  [' ', ' ', ' ']
+];
+
+let playerTurn = 'X';
+
+function printBoard() {
+  console.log('   0  1  2');
+  console.log('0 ' + board[0].join(' | '));
+  console.log('  ---------');
+  console.log('1 ' + board[1].join(' | '));
+  console.log('  ---------');
+  console.log('2 ' + board[2].join(' | '));
+}
+
+//my code below
+
+function horizontalWin() {
+  for(board[i] = 0; i < 3; i++) {
+    if (board[i] === ['X', 'X', 'X'] || ['O','O','O']) {
```

Comment:
cannot check equality that way you would have to check each element when you work with arrays. also, you cannot combine conditions like that with an || (OR)
```
if( (board[i][0] == 'X' && board[i][1] == 'X' && board[i][2] == 'X')  || 
     (board[i][0] == 'Y' && board[i][1] == 'Y' && board[i][2] == 'Y') ) {
...
```